Lib/test/test_struct.py

```diff
@@ -689,6 +689,17 @@ def test__struct_reference_cycle_cleaned_up(self):
         self.assertIsNone(
             module_ref(), "_struct module was not garbage collected")
 
+    def test__struct_types_immutable(self) -> None:
```

Comment:
We deliberately don't use typing in the stdlib, neither in public API's (which use, quote Steve Dower, anti-typing-patterns), nor in tests (except for the obvious typing tests).[^1]

Also, this is CPython specific.

```suggestion
    @support.cpython_only
    def test__struct_types_immutable(self):
```

[^1]: If you want to introduce typing in tests, please raise a discussion on Discourse and/or python-dev first.

Reply:
Fixed.